app/src/main/java/com/sopt/now/data/dto/request/RequestLoginDto.kt

```diff
@@ -0,0 +1,12 @@
+package com.sopt.now.data.dto.request
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class RequestLoginDto(
+    @SerialName("authenticationId")
+    val authenticationId: String,
+    @SerialName("password")
+    val password: String
+)
```

Comment:
DTO들을 잘 선언해주셨네요 👍 

이렇게 해도 문제는 없는데 하나의 팁을 더 드리면, `Serializable`을 사용하면서 data class에 default 값들을 넣어주면 나중에 서버에서 값이 null로 내려오거나 아무런 값도 내려오지 않았을 때 앱이 exception을 던지지 않고 Default 값을 대체해줘요. 그래서 default 값을 넣어주어도 좋은데 이거는 그냥 알아만 두시거나 다음 작업부터 반영해주셔도 될 것 같아요.

default 값을 넣게 되면 아래처럼 될거에요.
```suggestion
data class RequestLoginDto(
    @SerialName("authenticationId")
    val authenticationId: String = "",
    @SerialName("password")
    val password: String = ""
)
```

Reply:
헉 Default 값을 설정하면 이러한 장점이 있군요...! 감사합니다 적용해볼게요ㅎㅎ!